### PR TITLE
Fixed by adding a margin-right of 60px

### PR DESCRIPTION
**Pull request by Velythyl**

### CODE REVIEW

File: src/components/CohortBuilder/index.js [hunk 1]
```diff
@@ -114,6 +114,9 @@ const CohortBuilder = compose(
           .catch(console.error);
       };
 
+      const participantCountWidth = document.getElementsByClassName("participantsCountContainer").offsetWidth;
+      console.log("THE WIDTH:"+participantCountWidth)
```
Comment (nyanofthemoon) [inline src/components/CohortBuilder/index.js]: tiny cleanup? @Velythyl

File: src/components/CohortBuilder/index.js [hunk 2]
```diff
@@ -150,6 +157,7 @@ const CohortBuilder = compose(
             </SqonBuilderContainer>
           </FullWidthWhite>
 
+
```
Comment (nyanofthemoon) [inline src/components/CohortBuilder/index.js]: tiny cleanup? @Velythyl

File: src/components/CohortBuilder/index.js (hunk 1 above)
Comment (Velythyl) [reply src/components/CohortBuilder/index.js]: oooops forgot to remove that debugging

Comment (Velythyl) [reply src/components/CohortBuilder/index.js]: done!

File: src/components/CohortBuilder/index.js (hunk 2 above)
Comment (Velythyl) [reply src/components/CohortBuilder/index.js]: done!

File: src/components/CohortBuilder/index.js (hunk 1 above)
Comment (nyanofthemoon) [reply src/components/CohortBuilder/index.js]: Where do we use `participantCountWidth` ? @Velythyl

**Pull request merged**